src/fights.ts

```diff
@@ -701,6 +702,16 @@ function molemanReady() {
   return have($item`molehill mountain`) && !get("_molehillMountainUsed");
 }
 
+const stunDurations = new Map<Skill | Item, Delayed<number>>([
+  [$skill`Blood Bubble`, 1],
+  [$skill`Entangling Noodles`, () => (myClass() === $class`Pastamancer` ? 1 : 0)],
+  [$skill`Frost Bite`, 1],
+  [$skill`Shadow Noodles`, 2],
```

Comment:
Shadow Noodles cannot be used in the same fight as Entangling Noodles; I don't think(?) this handles that(?)